lib/common/grpc-service.js

```diff
@@ -266,6 +268,25 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
     request: function(_, onResponse) {
       respError = null;
 
+      if (stream) {
+        service[protoOpts.method](reqOpts, grpcOpts)
+          .pipe(stream)
```

Comment:
> Do you get an error about it?

Sure do!

```sh
Uncaught TypeError: Invalid non-string/buffer chunk
      at validChunk (node_modules/readable-stream/lib/_stream_writable.js:218:14)
      at DestroyableTransform.Writable.write (node_modules/readable-stream/lib/_stream_writable.js:239:53)
      at node_modules/stream-cache/lib/StreamCache.js:31:10
      at Array.forEach (native)
      at StreamCache.pipe (node_modules/stream-cache/lib/StreamCache.js:30:17)
      at onResponse (node_modules/retry-request/index.js:115:19)
      at ClientReadableStream.<anonymous> (lib/common/grpc-service.js:275:18)
      at ClientReadableStream._emitStatusIfDone (node_modules/grpc/src/node/src/client.js:187:10)
      at ClientReadableStream._readsDone (node_modules/grpc/src/node/src/client.js:158:8)
      at readCallback (node_modules/grpc/src/node/src/client.js:229:12)
```

> Also, is grpc returning a stream in object mode?

Sure is! Based on their docs, I'm not sure if it's configurable. https://github.com/grpc/grpc.github.io/blob/master/docs/tutorials/basic/node.md#streaming-rpcs